arch/arm/boot/dts/overlays/tc358743-overlay.dts

```diff
@@ -106,7 +106,7 @@
 	__overrides__ {
 		4lane = <0>, "-2+3-7+8";
 		link-frequency = <&tc358743_0>,"link-frequencies#0";
-		media-controller = <&csi>,"brcm,media-controller?";
+		media-controller = <0>,"=9";
```

Comment:
I think the correct change here is to delete fragment 9, and even then that's only cosmetic.